modules/rosa-quick-start-environment-setup.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * rosa_quick_start/rosa-quick-start.adoc
+
+[id="rosa-quick-start-environment-setup_{context}"]
+= Setting up the environment
+
+Before you create a {product-title} (ROSA) cluster, you must enable ROSA in your AWS account, install and configure the required CLI tools, and verify the configuration of the CLI tools.
+
+You must also verify that the AWS Elastic Load Balancing (ELB) service role exists and that the required AWS resource quotas are available.
```

Comment:
can we link or refer them to where the actual command is?  In case they dont know how, lets make it easy for them.

Reply:
Thanks for this suggestion. My view is that these two paragraphs are a short introduction outlining what is then described in the subsections for the section. The "Creating the ELB service role" and "Verifying AWS quota availability" subsections follow on from this. To link to those, we would have to remove this intro from a module and add it directly to the top level assembly.

To make sure that it is clear to the reader that we are describing the function of the subsections to follow, I will add something like "You can use the following procedures to..."

Reply:
I see what you meant.  Hmm, this darn linking issue...  Are you able to link to that other module from this module?